cmapPy/pandasGEXpress/concat_gctoo.py

```diff
@@ -261,16 +262,16 @@ def assemble_common_meta(common_meta_dfs, fields_to_remove):
         all_meta_df_sorted (pandas df)
 
     """
-    # Remove any column headers that will prevent dfs from being identical
-    for df in common_meta_dfs:
-        df.drop(fields_to_remove, axis=1, inplace=True)
-
     # Remove any column headers that are not present in all dfs (and sort)
     shared_column_headers = sorted(set.intersection(*[set(df.columns) for df in common_meta_dfs]))
-    common_meta_dfs = [df[shared_column_headers] for df in common_meta_dfs]
+    trimmed_common_meta_dfs = [df[shared_column_headers] for df in common_meta_dfs]
+
+    # Remove any column headers that will prevent dfs from being identical
```

Comment:
Ah, gotcha... forgot about that. Yeah, I think I would be in favor of consolidating the two into one, although reading the function names down the script is pretty funny.